doc/5_saving.md

```diff
@@ -97,6 +97,70 @@ val company = Company("DataStax", address)
 sc.parallelize(Seq(company)).saveToCassandra("test", "companies")
 ```
 
+## Specifying TTL and WRITETIME
+Spark Cassandra Connector saves the data without explicitly specifying TTL or WRITETIME. If a certain 
+values for them have to be used, there are a couple options provided by the API. 
+
+TTL and WRITETIME options are specified as properties of `WriteConf` object, which can be optionally 
+passed to `saveToCassandra` method. TTL and WRITETIME options are specified independently from one 
+another. 
+
+### Using a constant value for all rows
+When the same value should be used for all the rows, one can use the following syntax:
+
+```scala
```

Comment:
Add "import com.datastax.spark.connector.writer._" so examples will work in Spark shell